hilti/src/compiler/jit.cc

```diff
@@ -71,10 +71,7 @@ JIT::JIT(std::shared_ptr<Context> context) : _context(std::move(context)) {
     _jit = std::make_unique<detail::ClangJIT>(_context);
 }
 
-JIT::~JIT() {
-    if ( _jit )
-        finishRuntime();
-}
+JIT::~JIT() {}
```

Comment:
This does nothing now and isn't declared `virtual` either. I'd say let's just remove this custom destructor altogether.